### PR TITLE
tiup: add 3 docs cluster-rename/cluster-restart/dm-import

### PR DESCRIPTION
**Pull request by CharLotteiu**
### What is changed, added or deleted? (Required)

Add documents for commands `cluster rename`, `cluster restart`, and `dm import`.

Add 3 docs:

- tiup/tiup-component-cluster-rename.md
- tiup/tiup-component-cluster-restart.md
- tiup/tiup-component-dm-import.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4868
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by CharLotteiu** [comment]
Waiting for https://github.com/pingcap/docs/pull/5282 to be merged to fix dead links.

File: tiup/tiup-component-cluster-rename.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+title: tiup cluster rename
+---
+
+# tiup cluster rename
+
+The cluster name is specified when [the cluster is deployed](/tiup/tiup-component-cluster-deploy.md). If you want to change the cluster name after the cluster is deployed, you can use the command `tiup cluster rename`.
+
+> **Note:**
+>
+> If the tiup cluster is configured with the `dashboard_dir` field of `grafana_servers`, when you execute the command `tiup cluster rename` to rename the cluster, the following additional steps are required:
```
Comment (qiancai) [inline tiup/tiup-component-cluster-rename.md:11]:
```suggestion
> If the `dashboard_dir` field of `grafana_servers` is configured for the TiUP cluster, after you execute the command `tiup cluster rename` to rename the cluster, the following additional steps are required:
```

File: tiup/tiup-component-cluster-restart.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+- If this option is not specified, TiUP restarts all nodes by default.
+
+> **Note:**
+>
+> If the option `-R, --role` is specified at the same time, TiUP restarts service nodes that match both the requirements of `-N, --node` and `-R, --role`.
+
+### -R, --role
+
+- Specified roles of nodes to be restarted. The value of this option is a comma-separated list of the roles of the nodes. You can get the roles of the nodes from the second column of the [cluster status table](/tiup/tiup-component-cluster-display.md).
+- Data type: `STRINGS`
```
Comment (qiancai) [inline tiup/tiup-component-cluster-restart.md:38]:
```suggestion
- Data type: `STRING`
```

File: tiup/tiup-component-cluster-restart.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+`<cluster-name>`: the name of the cluster to operate on. If you forget the cluster name, you can check it with the [cluster list](/tiup/tiup-component-cluster-list.md) command.
+
+`<cluster-name>`
+
+## Options
+
+### -N, --node
+
+- Specifies the nodes to be restarted. The value of this option is a comma-separated list of node IDs. You can get the node ID from the first column of the [cluster status table](/tiup/tiup-component-cluster-display.md).
+- Data type: `STRINGS`
```
Comment (qiancai) [inline tiup/tiup-component-cluster-restart.md:28]:
```suggestion
- Data type: `STRING`
```

File: tiup/tiup-component-cluster-restart.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Syntax
+
+```shell
+tiup cluster restart <cluster-name> [flags]
+```
+
+`<cluster-name>`: the name of the cluster to operate on. If you forget the cluster name, you can check it with the [cluster list](/tiup/tiup-component-cluster-list.md) command.
+
+`<cluster-name>`
```
Comment (qiancai) [inline tiup/tiup-component-cluster-restart.md:21]: Is line 21 unneeded? If yes, please remove it.

File: tiup/tiup-component-cluster-restart.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+## Options
+
+### -N, --node
+
+- Specifies the nodes to be restarted. The value of this option is a comma-separated list of node IDs. You can get the node ID from the first column of the [cluster status table](/tiup/tiup-component-cluster-display.md).
```
Comment (qiancai) [inline tiup/tiup-component-cluster-restart.md:27]:
```suggestion
- Specifies the nodes to be restarted. The value of this option is a comma-separated list of node IDs. You can get the node IDs from the first column of the [cluster status table](/tiup/tiup-component-cluster-display.md).
```

File: tiup/tiup-component-cluster-restart.md (hunk 4 above)
Comment (CharLotteiu) [inline tiup/tiup-component-cluster-restart.md:21]:
```suggestion
```

File: tiup/tiup-component-cluster-restart.md (hunk 5 above)
Comment (CharLotteiu) [reply tiup/tiup-component-cluster-restart.md:27]:
```suggestion
- Specifies the nodes to be restarted. The value of this option is a comma-separated list of node IDs. You can get the node ID from the first column of the [cluster status table](/tiup/tiup-component-cluster-display.md) returned by the `tiup cluster display` command.
```

File: tiup/tiup-component-cluster-restart.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+- Data type: `STRINGS`
+- If this option is not specified, TiUP restarts all nodes by default.
+
+> **Note:**
+>
+> If the option `-R, --role` is specified at the same time, TiUP restarts service nodes that match both the requirements of `-N, --node` and `-R, --role`.
+
+### -R, --role
+
+- Specified roles of nodes to be restarted. The value of this option is a comma-separated list of the roles of the nodes. You can get the roles of the nodes from the second column of the [cluster status table](/tiup/tiup-component-cluster-display.md).
```
Comment (CharLotteiu) [inline tiup/tiup-component-cluster-restart.md:37]:
```suggestion
- Specified the roles of nodes to be restarted. The value of this option is a comma-separated list of the roles of the nodes. You can get the roles of the nodes from the second column of the [cluster status table](/tiup/tiup-component-cluster-display.md) returned by the `tiup cluster display` command.
```

**Pull request merged**